README.md

```diff
@@ -25,13 +18,8 @@ Coverage.jl
 
 *Step 1:* Navigate to your test directory, and start julia like this:
 ```sh
-julia --code-coverage=user
-```
-or, if you're running Julia 0.4 or higher,
-```sh
-julia --code-coverage=user --inline=no
+julia --code-coverage=tracefile-%p.info --code-coverage=user
```

Comment:
I don't understand this line. Quoting `julia --help` (versoin 1.1.0-DEV from `relase-1.1`):
```
 --code-coverage={none|user|all}, --code-coverage
                           Count executions of source lines (omitting setting is equivalent to "user")
```
So is the option to specify a trace file something new, which has not yet landed in `release-1.1`? If so, shouldn't this example be qualified, and explain a bit more? (Also, do I really *have* to specify a tracefile? And does this mean I can't name a tracefile "user"? This all seems a bit odd to me).

Reply:
Yes, the help for it will explain more when that gets added: https://github.com/JuliaLang/julia/blob/11ce9f3fbffe55e73bfec968d970b26f8e8e1e19/src/jloptions.c#L133-L134

> shouldn't this example be qualified

I suppose, I could. Given the age of some of the other content here, I figured it's better just to be concise.

Reply:
Well, but now any Julia 1.0 user trying to follow this example would get an error. Indeed, I just tried:
```
$ julia --code-coverage=tracefile-%p.info --code-coverage=user
ERROR: julia: invalid argument to --code-coverage (tracefile-%p.info)
```
As long as Julia 1.1 is not released (and I'd argue: even for quite a while after that), shouldn't the README also cate for Julia 1.0 users? So why not just
```
julia --code-coverage=user
```
as before? Is it even necessary to mention that `tracefile` at all by default? (Since I have no idea what the `tracefile` is for, and since this PR nowhere says, I can't tell).

Since you asked for my opinion: I would not merge it in this form if it was my project / my decision. As it is, though, of course you can do whatever you want.